packages/react/package.json

```diff
@@ -31,6 +31,7 @@
     "@babel/plugin-transform-react-inline-elements": "latest",
     "@babel/preset-env": "latest",
     "@babel/preset-react": "latest",
+    "@size-limit/preset-small-lib": "^2.1.0",
```

Comment:
why this change?

Reply:
In the `@latest` version of the library they've changed the setup to 3 different "types", it appears to be "Web app", "component/UI library" or "general library" - I think this best match is the latter.

You'll spot in Travis that since [this build](https://travis-ci.org/microlinkhq/sdk/builds/570504944) it was failing due to the output they show if you're still using `size-limit` w/ `2.1.0`.

![image](https://user-images.githubusercontent.com/5795227/63301098-02f85900-c2da-11e9-96f8-7194aadb170c.png)

I _do_ need to change the version to `latest` though, whoops 😅